feature/client/src/main/java/com/mifos/feature/client/clientDetails/presentation/ClientDetailsScreen.kt

```diff
@@ -0,0 +1,474 @@
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.asAndroidBitmap
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.core.content.FileProvider
+import androidx.hilt.navigation.compose.hiltViewModel
+import coil.compose.AsyncImage
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+import com.google.accompanist.permissions.isGranted
+import com.google.accompanist.permissions.rememberMultiplePermissionsState
+import com.mifos.core.common.utils.Utils
+import com.mifos.core.designsystem.component.MifosClientDetailsText
+import com.mifos.core.designsystem.component.MifosLoadingCircularProgress
+import com.mifos.core.designsystem.component.MifosLoanAccountExpendableCard
+import com.mifos.core.designsystem.component.MifosMenuDropDownItem
+import com.mifos.core.designsystem.component.MifosSavingsAccountExpendableCard
+import com.mifos.core.designsystem.component.MifosSelectImageDialog
+import com.mifos.core.designsystem.component.MifosSweetError
+import com.mifos.core.designsystem.theme.Black
+import com.mifos.core.designsystem.theme.BluePrimary
+import com.mifos.core.designsystem.theme.BluePrimaryDark
+import com.mifos.core.designsystem.theme.White
+import com.mifos.core.objects.accounts.loan.LoanAccount
+import com.mifos.core.objects.accounts.savings.DepositType
+import com.mifos.core.objects.accounts.savings.SavingsAccount
+import com.mifos.core.objects.client.Client
+import com.mifos.feature.client.R
+import java.io.File
+import java.util.Objects
+
+/**
+ * Created by Aditya Gupta on 18/03/24.
+ */
+
+@Composable
+fun ClientDetailsScreen(
+    clientId: Int,
+    onBackPressed: () -> Unit,
+    addLoanAccount: (Int) -> Unit,
+    addSavingsAccount: (Int) -> Unit,
+    charges: (Int) -> Unit,
+    documents: (Int) -> Unit,
+    identifiers: (Int) -> Unit,
+    moreClientInfo: (Int) -> Unit,
+    notes: (Int) -> Unit,
+    pinpointLocation: (Int) -> Unit,
+    survey: (Int) -> Unit,
+    uploadSignature: (Int) -> Unit,
+    loanAccountSelected: (Int) -> Unit,
+    savingsAccountSelected: (Int, DepositType) -> Unit,
+    activateClient: (Int) -> Unit
+) {
+
+    val clientDetailsViewModel: ClientDetailsViewModel = hiltViewModel()
```

Comment:
Inject hilt in function like 
val viewModel: : ClientDetailsViewModel = hiltViewModel()